### PR TITLE
README tweaks, proposal

### PR DESCRIPTION
**Pull request by jgehrcke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,16 +9,25 @@
 
 # The Open Source Observability Distribution
 
-Features that are not guarded behind a paywall.
```
Comment (jgehrcke) [inline README.md:12]: I think this is not the kind of language we want to have in our main README. Would love for us to discuss async maybe. I really propose to just remove it for now :).

File: README.md [hunk 2]
```diff
@@ -9,16 +9,25 @@
...
-Opstrace installs secure, horizontally-scalable, open source observability in your cloud account combining open APIs with reduced toil.
-Opstrace is:
+Opstrace is a secure, horizontally-scalable, open source observability platform installed in your cloud account.
```
Comment (jgehrcke) [inline README.md:12]: I found "Opstrace installs .. observability" to be semantically weird for a long time: one cannot install an abstract concept like observability, just like one cannot install happiness :-).

What do you think?

File: README.md [hunk 3]
```diff
@@ -9,16 +9,25 @@
...
+It combines open APIs with reduced toil.
+
+Highlights:
```
Comment (jgehrcke) [inline README.md:15]: To not have "Opstrace is.." twice, I have replaced this one here with "Highlights", which I think what this list really is supposed to be!

File: README.md [hunk 4]
```diff
@@ -9,16 +9,25 @@
...
 
 * Horizontally scalable for ingestion and queries.
 * Durable with inexpensive long-term retention.
-* Rigoriously tested end-to-end, freqently, for confident upgrades.
```
Comment (jgehrcke) [inline README.md:18]: there was a typo here in `freqently`.

File: README.md [hunk 5]
```diff
@@ -9,16 +9,25 @@
...
+* Rigoriously tested end-to-end.
+* Large efforts towards confident upgrades.
```
Comment (jgehrcke) [inline README.md:20]: I find it important to decouple the points of rigorous end-to-end testing, and the effort towards seamless upgrades.

File: README.md [hunk 6]
```diff
@@ -9,16 +9,25 @@
...
 * Secure by default with TLS and authenticated endpoints.
-* Easy to configure with new web interfaces and APIs.
-* All the open source projects you know and love in one place.
+* Easy to configure with GUIs and APIs.
```
Comment (jgehrcke) [inline README.md:22]: 'web interface' is I think actually GUI, right?

File: README.md [hunk 7]
```diff
@@ -9,16 +9,25 @@
...
+
+We walk on the shoulders of giants; Opstrace contains all the open source projects you know and love in one place:
+* [Cortex](https://github.com/cortexproject/cortex)
+* [Loki](https://github.com/grafana/loki)
+* [Prometheus](https://github.com/prometheus/prometheus)
+* [Kubernetes](https://github.com/kubernetes/kubernetes)
+* and many more!
```
Comment (jgehrcke) [inline README.md:29]: just an idea, please share your opinion

File: README.md [hunk 8]
```diff
@@ -81,7 +91,7 @@ By participating in this project, you agree to abide by its terms.
 
 At Opstrace, we host a service that automatically provisions subdomains like `<my_name>.opstrace.io`.
 This makes it easy and clean to use the URLs for both humans and machines.
-To accomplish this safely, we require login via oauth subject to our [privacy poicy](https://go.opstrace.com/privacy-policy).
+To accomplish this safely, we require login via Auth0 subject to our [privacy poicy](https://go.opstrace.com/privacy-policy).
```
Comment (jgehrcke) [inline README.md:94]: Haha, "oauth" is not a thing! : )

File: README.md (hunk 1 above)
Comment (triclambert) [reply README.md:12]: That's fine.

File: README.md (hunk 2 above)
Comment (triclambert) [reply README.md:12]: 👍

File: README.md (hunk 7 above)
Comment (triclambert) [reply README.md:29]: Looks great.

Comment (spahl) [reply README.md:29]: +1

**Review by spahl — APPROVED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (spahl) [reply README.md:12]: Same. The idea is to be clear that this project will only be open:-)

Comment (jgehrcke) [reply README.md:12]: OK, we'll find better wording!

**Pull request merged**